docs/1.3_to_1.4_migration_guide.md

```diff
@@ -1,15 +1,25 @@
-# List of broken changes for migration from indy-node 1.3 to 1.4:
+# List of broken changes for migration from indy-node 1.3 to 1.4
 
 -**Another format of REPLY for "WRITE" transactions**. New REPLY format you can see in:
  [REPLY for "WRITE" txns](https://github.com/hyperledger/indy-node/blob/master/docs/requests.md#reply-structure-for-write-requests).
  The main goal for this changes is to divide data and metadata.
 
--**Change key of ReqIdrToTxn store**.   For now, ReqIdrToTxn store request from client as a map of:
+-**Change key of ReqIdrToTxn stores**.   For now, ReqIdrToTxn store request from client as a map of:
```

Comment:
It should be
`**Change key of ReqIdrToTxn store**.   For now, ReqIdrToTxn stores .....`